cloup/formatting/sep.py

```diff
@@ -206,9 +208,10 @@ def __init__(self, pattern: str):
 
     def __call__(self, width: int) -> str:
         pattern = self.pattern
-        if len(pattern) == 1:
+        pattern_len = width(pattern)
```

Comment:
Here we go: collision.

Reply:
Yeah, fair. What name would you prefer?

Reply:
Maybe `display_len` or `display_width`?

Reply:
I tend to append `_of` when there's a change of using the same identifier for a variable: `display_width_of(line: str)`

Reply:
@janluke Why not just call it `display_width` as long as there's no existing function or variable of that name? (I'm pretty sure there's not.) Making the names of (fairly frequently used) functions too long might be a bit annoying, no?

Reply:
Also, how do I include the package dependency, may I ask?

Reply:
> Why not just call it display_width as long as there's no existing function or variable of that name?

Three chars don't make much difference to me but go for `display_width` if you strongly prefer it.

> Also, how do I include the package dependency, may I ask?

Add the dependency to setup.py. You can ignore the requirements folder.

Reply:
@janluke Oh right. I was getting confused by the presence of setup.cfg and especially tox.ini (which I'm not really familiar with). Will do!

Reply:
Re the names, sounds fair. Are you happy with `unstyled_display_width` too? Or `display_width_unstyled`?

Reply:
@janluke Just pushed, but let me know about the above...